RecoLocalCalo/EcalRecProducers/test/testEcalUncalibRecHitPhase2WeightsProducer.py

```diff
@@ -0,0 +1,189 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Phase2C10_cff import Phase2C10
+from Configuration.Eras.Modifier_phase2_ecal_devel_cff import phase2_ecal_devel
+
+from RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2_cff import *
+from RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi import *
+
+process = cms.Process('RECO',Phase2C10,phase2_ecal_devel)
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('SimGeneral.MixingModule.mixNoPU_cfi')
+process.load('Configuration.Geometry.GeometryExtended2026D60Reco_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('Configuration.StandardSequences.RawToDigi_cff')
+process.load('Configuration.StandardSequences.L1Reco_cff')
+process.load('Configuration.StandardSequences.Reconstruction_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+process.load('RecoLocalCalo.EcalRecProducers.ecalUncalibRecHitPhase2_cff')
+process.load('RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi')
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10),
+    output = cms.optional.untracked.allowed(cms.int32,cms.PSet)
+)
+
+# Input source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/afs/cern.ch/work/b/bpinolin/public/EcalPh2/step2.root'),
```

Comment:
please do not use a private path
a group location could be OK

Reply:
Ok, I will use a group location